<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configOptions</class>
 <widget class="QDialog" name="configOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>General Options</string>
    </attribute>
    <widget class="QPushButton" name="pbSelectLanguage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>370</y>
       <width>331</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Language</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>531</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>HiDPI Settings</string>
     </property>
     <widget class="QCheckBox" name="cbHiDPI">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable HiDPI Scalling</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditHiDPI">
      <property name="geometry">
       <rect>
        <x>102</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale Factor
(Enter A Decimal Number)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>SiriKali Will Have To Be Restarted For These Changes To Take Effect.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="cbAutoOpenMountPoint">
        <property name="text">
         <string>Auto Open Mount Point</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbReUseMountPoint">
        <property name="text">
         <string>Re Use Mount Point</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="chShowDebugWindowOnStartup">
        <property name="text">
         <string>Show Debug Window On Startup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="cbStartMinimized">
        <property name="text">
         <string>Start Minimized</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="cbInternallyManageBackEnds">
        <property name="text">
         <string>Manage Cryfs/Securefs/Gocryptfs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>331</width>
       <height>34</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QComboBox" name="cbThemes">
        <item>
         <property name="text">
          <string>Platform Default Theme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dark Theme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light Theme</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget5_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>331</width>
       <height>34</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QCheckBox" name="cbShowTrayIcon">
        <property name="text">
         <string>Show Tray Icon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="setExecutables">
    <attribute name="title">
     <string>External Commands</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set File Manager</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditExecutableKeySource">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMountPointPrefix">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFileManager">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Mount Point Prefix</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMountPointPrefix">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>30</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set External Plugin Executable</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Command To Run After Mount</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAfterMountCommand">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditBeforesUnMount">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Command To Run Before UnMount</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbPostMountCommand">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>210</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbPreUnMountCommand">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>270</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Command To Run Periodically</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRunPeriodically">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRunPeriodicallyCommand">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>330</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRunPeriodicallyInterval">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>370</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>370</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval In Minutes Periodic Command Will Run</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSetFileManager">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>90</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSetExternalCommand">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>150</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbExternalExecutableDefault">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSetFileManagerToDefault">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMountPointPrefixDefault">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRunPostMountCommandDefault">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRunPreUnMountCommandDefault">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>270</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRunCommandPeriodicallyDefault">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>330</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cbAutoOpenMountPoint</tabstop>
  <tabstop>cbReUseMountPoint</tabstop>
  <tabstop>cbStartMinimized</tabstop>
  <tabstop>pbSelectLanguage</tabstop>
  <tabstop>lineEditMountPointPrefix</tabstop>
  <tabstop>pbMountPointPrefix</tabstop>
  <tabstop>pbMountPointPrefixDefault</tabstop>
  <tabstop>lineEditFileManager</tabstop>
  <tabstop>pbSetFileManager</tabstop>
  <tabstop>pbSetFileManagerToDefault</tabstop>
  <tabstop>lineEditExecutableKeySource</tabstop>
  <tabstop>pbSetExternalCommand</tabstop>
  <tabstop>pbExternalExecutableDefault</tabstop>
  <tabstop>lineEditAfterMountCommand</tabstop>
  <tabstop>pbPostMountCommand</tabstop>
  <tabstop>pbRunPostMountCommandDefault</tabstop>
  <tabstop>lineEditBeforesUnMount</tabstop>
  <tabstop>pbPreUnMountCommand</tabstop>
  <tabstop>pbRunPreUnMountCommandDefault</tabstop>
  <tabstop>lineEditRunPeriodically</tabstop>
  <tabstop>pbRunPeriodicallyCommand</tabstop>
  <tabstop>pbRunCommandPeriodicallyDefault</tabstop>
  <tabstop>lineEditRunPeriodicallyInterval</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
